src/s2/s2builderutil_closed_set_normalizer_test.cc

```diff
@@ -71,22 +71,22 @@ class NormalizeTest : public testing::Test {
                      DuplicateEdges::KEEP, SiblingPairs::KEEP));
   }
 
-  void Run(const string& input_str, const string& expected_str);
+  void Run(const std::string& input_str, const std::string& expected_str);
```

Comment:
The way `s2geometry` likes to do this in `.cc` files is to do `using std::string;` after the `#include`s, then leave plain `string` in the rest of the file. I can merge it if you do that.

`.h` is good like you have it.